src/Core/ServiceWrapper/CLI/CliOption.cs

```diff
@@ -0,0 +1,65 @@
+﻿using CommandLine;
+using System;
+using System.ComponentModel;
+using System.Diagnostics;
+using System.IO;
+using winsw.Native;
+using WMI;
+
+namespace winsw.CLI
+{
+    public abstract class CliOption
```

Comment:
This class contains not only an option, but also the execution implementation. I would recommend calling it `CLICommand`, and also renaming other classes in a similar way